kafka/pom.xml

```diff
@@ -31,6 +31,38 @@
     <basepom.test.timeout>120</basepom.test.timeout>
   </properties>
 
+  <build>
+  <plugins>
+    <plugin>
+      <groupId>org.apache.maven.plugins</groupId>
+      <artifactId>maven-shade-plugin</artifactId>
+      <executions>
+        <execution>
+          <id>shade-kafka</id>
+          <phase>package</phase>
+          <goals>
+            <goal>shade</goal>
+          </goals>
+        </execution>
+      </executions>
+      <configuration>
+        <shadedArtifactAttached>false</shadedArtifactAttached>
+        <artifactSet>
+          <includes>
+            <include>org.apache.kafka</include>
+          </includes>
+        </artifactSet>
+        <relocations>
+          <relocation>
+            <pattern>org.apache.kafka</pattern>
+            <shadedPattern>otj.logging.kafka</shadedPattern>
+          </relocation>
+        </relocations>
+      </configuration>
+    </plugin>
+  </plugins>
+  </build>
+
```

Comment:
We need to do better than that.

At least follow this feed, perhaps.
https://github.com/apache/kafka/releases.atom

Are you on their mailing list?